### ISSUE TITLE
Is it possible to export links?

### ISSUE THREAD

**Issue by viktorix** [shlinkio/shlink#1085]
I saw there is an import command, but I'm not seeing an export command. Is there no way to export? We're migrating Shlink from one server, which uses classic server setup, to a server using Docker image. So I wanted to export links and import them into the new instance. 

If there's no export command, I'm guessing we'll just need to migrate database.

Thanks.

**Comment by acelaya** [comment]
I'm afraid exporting is not yet supported, but the plan is to allow it at some point.

However, next Shlink version (2.7) will allow importing from another Shlink instance https://shlink.io/documentation/advanced/import-short-urls/#shlink-instance

If you are not in a hurry and can afford to wait a couple of weeks for it to be released, it will probably simplify things.

**Comment by viktorix** [comment]
@acelaya thanks for the quick reply. We have to migrate before the 14th, so can't wait. However, I did figure out a "quick" way to get all the links. I went to the "List short URLs" page in the web UI app. Selected "All short URLs". And then simply selected and copied each page into a spreadsheet. Since it uses HTML tables, it pasted everything nicely into individual cells. Took me 30 seconds to do a bunch of pages. Just FYI, in case anyone needs it before the export command is available.
